### PR TITLE
add a "consider flossing" day of the week phrase

### PR DESCRIPTION
**Pull request by twentylemon**
##### Summary

Adding a day of the week phrase. :floss:, yo

Change was made in browser, so I'm letting actions run `pytest` for me.

##### Checklist

- [x] this is a source code change
  - [ ] run `format` in the repository root
  - [ ] run `pytest` in the repository root
  - [x] link to issue being fixed using a [_fixes keyword_](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), if such an issue exists
  - [x] update the wiki documentation if necessary
- [ ] or, this is **not** a source code change

### CODE REVIEW

**Comment by twentylemon** [comment]
Actions keep failing due to missing nltk data? It's downloaded as part of the installation, not sure what's going on there.

File: duckbot/cogs/announce_day/phrases.py [hunk 1]
```diff
@@ -118,6 +118,7 @@
     "I take no pleasure in announcing that today is {today}, for I am a robot.",
     "I take no displeasure in announcing that today is {today}, for I am a robot.",
     "I'm glad I got turned on today, for it is {today}.",
+    "It's {today}. Also, consider flossing today.",
```
Comment (jameshughes89) [inline duckbot/cogs/announce_day/phrases.py]: "Also, consider flossing today." Feels a little abrupt. Perhaps "One may consider flossing on such a day", or something more flowery as that feels a little more on brand for duckbot.

Comment (twentylemon) [reply duckbot/cogs/announce_day/phrases.py]: Updated, also added a rude phrase for kicks.

**Comment by twentylemon** [comment]
Tests should succeed after https://github.com/duck-dynasty/duckbot/pull/565 is merged.

**Review by jameshughes89 — APPROVED** [review]
Looks good, you dunks.

**Pull request merged**